runtime/lua/vim/_system.lua

```diff
@@ -245,6 +245,10 @@ local function spawn(cmd, opts, on_exit, on_error)
   local handle, pid_or_err = uv.spawn(cmd, opts, on_exit)
   if not handle then
     on_error()
+    if type(pid_or_err) == 'string' and (pid_or_err:match('ENOENT') or pid_or_err:match('EACCES')) then
```

Comment:
Nit:
```suggestion
    if string.match(pid_or_err, 'ENOENT') or string.match(pid_or_err, 'EACCES') then
```

`string.match` will return `nil` if `pid_or_err` isn't a string.

Reply:
that works for a number, but not boolean/table. but this check is also for LuaLS